activestate.yaml

```diff
@@ -47,14 +47,35 @@ constants:
 scripts:
   - name: install-deps
     language: bash
+    if: ne .Shell "cmd"
     value: |
-      export GO111MODULE=on
-      goflags="${GOFLAGS}"; unset GOFLAGS
-      GOFLAGS="${goflags}"
-      pushd ~
-      go install github.com/ActiveState/gozip/cmd/gozip@e455986
-      popd
-      $scripts.install-deps-os
+      oldGOFLAGS="${GOFLAGS}"
+      # strip mod flag in CI only
+      if [ -z ${CI+x} ]; then
+        export GOFLAGS="$(echo ${GOFLAGS} | sed 's#-mod=vendor \?##')"
+      fi
```

Comment:
Assuming this is necessary for Go 1.20; please add a comment explaining why we're doing this.

Reply:
I'm not exactly sure what change occurred to require this, but it is a known issue: https://github.com/golang/go/issues/45811

IIUC, we can vendor the source needed to build these executables, then ensure that the vendored source is used by setting the mod flag to =vendor.

I've created a new story to look into our install-deps setup: https://activestatef.atlassian.net/browse/DX-1673

Reply:
Can't we just drop -mod=vendor? I thought the recent Go versions use the vendor dir if it exists, no need for the flag?